bpf/node_config.h

```diff
@@ -120,7 +120,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define MONITOR_AGGREGATION 5
 #define MTU 1500
 #define EPHEMERAL_MIN 32768
-#if defined(ENABLE_NODEPORT) || defined(ENABLE_HOST_FIREWALL)
+#if defined(ENABLE_NODEPORT) || defined(ENABLE_HOST_FIREWALL) || defined(ENABLE_NAT46)
```

Comment:
Did this work before because we had never enabled `ENABLE_NAT46` without the other two being disabled?

Reply:
I think it worked because we were only enabling it for bpf_xxx.c files which had `#define CONNTRACK` hardcoded in their corresponding header file.